src/views/SignUp/utils.js

```diff
@@ -0,0 +1,102 @@
+import firebase, { firestore } from '../../firebase';
```

Comment:
I think all these functions should be in the api file. That way we have one place the concentrates all calls to firestore

Reply:
Agree, i'll move them